pkg/visor/visorconfig/parse.go

```diff
@@ -38,20 +36,20 @@ func Parse(log *logging.MasterLogger, path string, raw []byte) (*V1, error) {
 	case V111Name:
 		fallthrough
 	case V110Name:
-		return parseV1(cc, raw)
+		return parseV1(cc, raw, hypervisor, services)
 	case V101Name:
-		return parseV1(cc, raw)
+		return parseV1(cc, raw, hypervisor, services)
 	case V100Name:
-		return parseV1(cc, raw)
+		return parseV1(cc, raw, hypervisor, services)
 	case V0Name, V0NameOldFormat, "":
```

Comment:
We should remove this case altogether if we are getting rid of `parseV0`.

Reply:
done